constants.py

```diff
@@ -6,6 +6,14 @@
 
 DB_NAME = "image_generation_service"
 
+MODEL_CLASSIFICATION_URL = "https://superensembleclassifier-production.up.railway.app/classify"
```

Comment:
should we have this variable as an environment variable

Reply:
if it's not defined in env variables, just ignore this step and return random model name, wdyt?